src/stactools/noaa_sst/constants.py

```diff
@@ -0,0 +1,27 @@
+from datetime import datetime
+from pyproj import CRS
+from pystac import Provider, ProviderRole, Link
+
+NOAA_SST_ID = "noaa-sst"
+SST_EPSG = 4326
+SST_CRS = CRS.from_epsg(SST_EPSG)
+LICENSE = "proprietary"
+lic_link = "https://github.com/stactools-packages/noaa-sst/tree/main/src/stactools/noaa_sst/license"
+"/data-license.txt"
```

Comment:
This doesn't continue the line, so `lic_link` becomes: `https://github.com/stactools-packages/noaa-sst/tree/main/src/stactools/noaa_sst/license`

Reply:
Fixed